packages/strapi-plugin-upload/middlewares/upload/index.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+const { join } = require('path');
+const range = require('koa-range');
+const koaStatic = require('koa-static');
+
+module.exports = strapi => ({
+  initialize() {
+    const staticDir = join(
+      strapi.dir,
+      strapi.config.middleware.settings.public.path || strapi.config.paths.static
```

Comment:
Why are there 2 possible paths ? Is it possible ton only have one ?

Reply:
When we will change the config this will change. right now I'm just keeping compatibility with the public middleware.